artifacts/snip-install-helm-cli-k8s.adoc

```diff
@@ -0,0 +1,56 @@
+[id="snip-install-helm-cli-k8s_{context}"]
```

Comment:
This is a procedure.

Reply:
I could not reuse the module twice without build errors. Maybe due to the old practice of setting the context in the title? This was the only way I could get the build to come out clean, but I would like to investigate that content reuse issue in the future.